### PR TITLE
Project cleanup Part 2

### PR DESCRIPTION
**Pull request by sungam3r**
Convert to file-scoped namespaces + unify csproj files. See with `hide whitespaces` option enabled.

### CODE REVIEW

**Comment by ArieGato** [comment]
@sungam3r I created PR https://github.com/ArieGato/serilog-sinks-rabbitmq/pull/142 which changes a lot of files. I really want to push that PR. It will probably result in a lot of conflicts.

**Comment by sungam3r** [comment]
Go on with #142, I'll resolve conflicts.

**Comment by sungam3r** [comment]
@ArieGato I resolved conflicts.

**Review by ArieGato — DISMISSED** [review]
Looks good. Thnx for the contribution.

File: Directory.Build.props [hunk 1]
```diff
@@ -3,14 +3,30 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <ImplicitUsings>enable</ImplicitUsings>
+    <Copyright>Copyright © Serilog Contributors 2019-2024</Copyright>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
```
[the remark this answers is not in the document]
Comment (sungam3r) [reply Directory.Build.props:10]: For csproj files.

**Pull request merged**